webdriver-spec.html

```diff
@@ -4064,13 +4064,12 @@
 
  <li><p>Let <var>result</var> be an empty JSON <a>List</a>.
 
- <li><p>For each <var>element</var> in <var>elements returned</var>:
-
-  <ul>
-   <li><p>Append to <var>result</var>
-    the <a data-lt="JSON Serialization of an element">JSON Serialization</a>
-    of the <var>element</var>.
-  </ul>
+ <li><p>For each <var>element</var> in <var>elements returned</var>,
```

Comment:
By this point `elements returned` will be a `NodeList` so we don't need to any other error checks. we just need to append. What situations would not return a `NodeList`?